lib/hyrax/publisher.rb

```diff
@@ -155,6 +155,10 @@ class Publisher
     # @macro a_registered_event
     register_event('object.acl.updated')
 
+    # @since 3.4.0
+    # @macro a_registered_event
+    register_event('object.acl.deleted')
```

Comment:
is it worth documenting that this is a delete of the whole ACL resource, not just of a single permission statement?